### PR TITLE
Fix imports on master

### LINKED ISSUE #37: fails to build: incorrect usage of internal package
```
go get gopkg.in/src-d/enry.v1/...
package gopkg.in/src-d/enry.v1/internal/code-generator
	imports gopkg.in/src-d/simple-linguist.v1/internal/code-generator/generator: use of internal package not allowed
```

```
go get gopkg.in/src-d/enry.v1/cli/enry
package gopkg.in/src-d/enry.v1/cli/enry: cannot find package "gopkg.in/src-d/enry.v1/cli/enry" in any of:
	/usr/lib/go-1.8/src/gopkg.in/src-d/enry.v1/cli/enry (from $GOROOT)
	/home/sourced/Projects/ast2vec/enry/src/gopkg.in/src-d/enry.v1/cli/enry (from $GOPATH)
```

This works after symlinking `github.com/src-d/enry` to `gopkg.in/src-d/enry.v1`
```
go get github.com/src-d/enry/cli/enry
```

### PR DESCRIPTION
**Pull request by bzz**
This is an attempt to make `enry` go'gettable again and resolve #37  

 - change imports from `gopkg.in/src-d/enry.v1/internal/tokenizer` to `github.com/src-d/enry/internal/tokenizer`
 - re-generate internal data structures from latest linguist

\cc @ajnavarro @smola @mcuadros for feedback on whether it's the right approach of fixing it in Golang

### CODE REVIEW

**Comment by bzz** [comment]
@mcarmonaa `make test` fails like

```
    --- FAIL: TestSimpleLinguistTestSuite/TestLinguistCorpus (2.52s)
        Error Trace:    common_test.go:383
    	            	path.go:351
    	            	path.go:376
    	            	path.go:376
    	            	path.go:398
    	            	common_test.go:387
    	Error:      	Not equal:
    	            	expected: "Perl 6"
    	            	actual: "Perl6"
    	Messages:   	01-dash-uppercase-i.t	expected: Perl 6	obtained: Perl6	status: failed
```

Are there any test fixtures or something that needs to be re-generated?

File: .travis.yml [hunk 1]
```diff
@@ -9,10 +9,6 @@ matrix:
     - go: tip
 
 install:
-  - rm -rf $GOPATH/src/gopkg.in/src-d
```
Comment (mcuadros) [inline .travis.yml]: why are you removing this? if you remove this lines the tests are being executed in a different path and testing the current master branch.

Comment (bzz) [inline .travis.yml]: Yes, that is the whole purpose of this PR - switch all things to master. See 4f42d95

If that is not the right thing to do in Golang, let's use this as disucssion\eduction point on how gopkg.in must be used.

**Comment by mcuadros** [comment]
We using gopkg.in for semantic versioning in all our golang projects.

**Comment by bzz** [comment]
> We using gopkg.in for semantic versioning in all our golang projects.

That was very helpful, thank you for pointing it out.

But is there any chance you could also please elaborate, for those of us who are new to Golang, how to solve the problem that I mention above in this situation?

**Comment by bzz** [comment]
As @mcuadros pointed out later, #37 is due to repository URL change and should be fixed by https://github.com/src-d/enry/releases/tag/v1.2.1

Still, details of go release policy with regarding to semantic versioning and/or imports are not clear.

**Pull request closed without merging**